src/app/models/Cake.ts

```diff
@@ -0,0 +1,35 @@
+const enum CakeStatus {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file named `cake-status.enum.ts`.